### DISCUSSION TITLE
Problem with generating APK file

### DISCUSSION THREAD

**Discussion by narkul** [IbrahimEssakine/Smart-Home#23] [Q&A]
Hello! I am new in Android development and was interested in your app. I tried to build APK file in Android Studio, but I am getting some errors:
`Error: Houses_Popup_Dialog must extend android.app.Activity [Instantiatable]
            android:name=".Houses_Popup_Dialog"`
`Error: Dialog_Test must extend android.app.Activity [Instantiatable]
            android:name=".Dialog_Test"`
Can you share with APK file?

**Comment by Mouad4399** [comment]
I added the APK file. Hope it will work for you, noting that the app is not fully functional as it is under development and may throw errors.
